### DISCUSSION TITLE
Cleanup 'old' issues...

### DISCUSSION THREAD

**Discussion by f-laurens** [nabaztag2018/pynab#240] [General]
It would be useful to review old issues that may be closed, either because they are obsolete/not relevant any more, or because they have been resolved by commits or PRs.

**Comment by f-laurens** [comment]
My candidate list of such issues:

- ~~#8 : not relevant anymore?~~
- #72: relevant? (users can customize this easily)
- ~~#104: resolved by #125~~
- ~~#129: still applicable?~~
- ~~#146: resolved by #269~~
- ~~#153: obsoleted by merge of #125?~~

**Reply by mehdichaouch** [reply]
@nabaztag2018 spring cleanup is coming 😃

**Reply by omevel** [reply]
I closed a couple of them :) (left #72 and #146)

**Reply by f-laurens** [reply]
A definite progress: all open issues now fit on a single page... ;-)

**Comment by f-laurens** [comment]
Looks like this discussion may be closed.
